wporg-gp-translation-events.php

```diff
@@ -84,6 +84,7 @@ public function gp_init() {
 		GP::$router->add( '/events/attend/(\d+)', array( 'Wporg\TranslationEvents\Routes\User\Attend_Event_Route', 'handle' ), 'post' );
 		GP::$router->add( '/events/host/(\d+)/(\d+)', array( 'Wporg\TranslationEvents\Routes\User\Host_Event_Route', 'handle' ), 'post' );
 		GP::$router->add( '/events/my-events', array( 'Wporg\TranslationEvents\Routes\User\My_Events_Route', 'handle' ) );
+		GP::$router->add( '/events/([a-z0-9_-]+)/([a-z0-9_-]+)(?:/([a-z0-9_-]+))?', array( 'Wporg\TranslationEvents\Routes\Event\Translations_Route', 'handle' ) );
```

Comment:
Could we "namespace" this route? Something like `/events/$slug/translations/$locale`. Would be consistent with other routes, and would make explicit that this is the translations page.

Reply:
Agreed, thanks for the suggestion! I've added variables.

Reply:
What I meant in the suggestion above was to add the literal string `translations` (as an example, can be another string) because the URL is currently simply `/events/wordcamp-bilbao/pt/`. Instead the URL would become `/events/wordcamp-bilbao/translations/pt/`.

I think this would make it easier to add routes in the future, without them clashing with the translations route. For example, if we decide to add a separate route for the list of attendees of an event, its URL would then be `/events/wordcamp-bilbao/attendees/`.